src/cliques.jl

```diff
@@ -37,24 +40,32 @@ function pothen_sun{T<:Integer}(par::Vector{T}, post::Vector{T}, colcount::Vecto
     ns, flag
 end
 
-function chordal_embedding{Tv<:Number,Ti<:Int}(A::SparseMatrixCSC{Tv,Ti}, Perm::Vector{Ti})
+function round_array(T, A)
+    if T <: Integer
+        map(x->Base.round(T,x),A)
+    else 
+        map(x->Base.round(x),A)
+    end
+end
+
+function chordal_embedding(A::SparseMatrixCSC{Tv,Ti}, Perm::Vector{Ti}) where {Tv<:Number,Ti<:Int}
     m, n = size(A)
-    S = CHOLMOD.Sparse(round(Float64,A))
+    S = CHOLMOD.Sparse(round_array(Float64,A))
```

Comment:
This should be 
`S= SuiteSparse.CHOLMOD.Sparse(round_array(Float64,A)`

Reply:
I believe this is taken care of by the "using SuiteSparse.CHOLMOD" earlier in the file. The line executes for me, anyway. If it is a matter of style, I defer to your preferences.

Reply:
Right. I missed that. Great point.